### PR TITLE
Document information source for prescriptions

### LINKED ISSUE #33389: document in README.md what information source we use to generate prescriptions
**Is your feature request related to a problem? Please describe.**
SSIA

**Describe the solution you'd like**
tbd

**Describe alternatives you've considered**
tbd

**Additional context**
tbd

### PR DESCRIPTION
**Pull request by mayaCostantini**
## Related Issues and Dependencies

Fixes #33389 

## This introduces a breaking change

- No 

## This should yield a new module release

- No

## This Pull Request implements

Document sources used to compute automatically generated prescriptions.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -43,6 +43,21 @@ Need help with a prescription?
...
 
+Data sources used for automatically generated prescriptions
+===========================================================
+
+Currently implemented `handlers
+<https://github.com/thoth-station/prescriptions-refresh-job/tree/master/thoth/prescriptions_refresh/handlers>`__
+in Thoth's weekly cronjob allow to auto-generate prescriptions for the given data:
+
+- CVE present in a package, from the `PyPA advisory-database
+<https://github.com/pypa/advisory-database>`__
```
Comment (harshad16) [inline README.rst:54]:
```suggestion
- CVE present in a package, from the `PyPA advisory-database <https://github.com/pypa/advisory-database>`_
```

File: README.rst [hunk 2]
```diff
@@ -43,6 +43,21 @@ Need help with a prescription?
...
+- Project maintenance and development practices as evaluated by the `OSSF Security Scorecards
+<https://github.com/ossf/scorecard>___
```
Comment (harshad16) [inline README.rst:56]:
```suggestion
- Project maintenance and development practices as evaluated by the `OSSF Security Scorecards <https://github.com/ossf/scorecard>`_
```

File: README.rst [hunk 3]
```diff
@@ -43,6 +43,21 @@ Need help with a prescription?
...
+- Information on package maintainance obtained via the GitHub API: if the given project is marked as archived, is forked from another project, hosts release notes, its number of maintainers, stars, contributors.
+- The package size, number of downloads, maintainers and last release date from `PyPI
+<https://pypi.org/>`___.
```
Comment (harshad16) [inline README.rst:59]:
```suggestion
- The package size, number of downloads, maintainers, and last release date from `PyPI <https://pypi.org/>`_.
```

**Review by harshad16 — CHANGES_REQUESTED** [review]
lgtm
just some fix is needed for formatting, to render as RST.

File: README.rst [hunk 4]
```diff
@@ -43,6 +43,21 @@ Need help with a prescription?
...
+===========================================================
+
+Currently implemented `handlers
+<https://github.com/thoth-station/prescriptions-refresh-job/tree/master/thoth/prescriptions_refresh/handlers>`_
+in Thoth's weekly cronjob allow to auto-generate prescriptions for the given data:
+
+- CVE present in a package, from the `PyPA advisory-database
+<https://github.com/pypa/advisory-database>`_
+- Project maintenance and development practices as evaluated by the `OSSF Security Scorecards
+<https://github.com/ossf/scorecard>`_
```
Comment (harshad16) [inline README.rst:56]:
```suggestion
- CVE present in a package, from the `PyPA advisory-database <https://github.com/pypa/advisory-database>`_
- Project maintenance and development practices as evaluated by the `OSSF Security Scorecards <https://github.com/ossf/scorecard>`_
```

**Review by harshad16 — CHANGES_REQUESTED** [review]
Thanks for the changes.
Spacing is mixing between points.

File: README.rst [hunk 5]
```diff
@@ -43,6 +43,19 @@ Need help with a prescription?
...
+
+Currently implemented `handlers
+<https://github.com/thoth-station/prescriptions-refresh-job/tree/master/thoth/prescriptions_refresh/handlers>`_
+in Thoth's weekly cronjob allow to auto-generate prescriptions for the given data:
+
+- CVE present in a package, from the `PyPA advisory-database <https://github.com/pypa/advisory-database>`_
+- Project maintenance and development practices as evaluated by the `OSSF Security Scorecards <https://github.com/ossf/scorecard>`_
+- Information on package maintainance obtained via the GitHub API: if the given project is marked as archived, is forked from another project, hosts release notes, its number of maintainers, stars, contributors.
+- The package size, number of downloads, maintainers and last release date from `PyPI
+<https://pypi.org/>`_.
```
Comment (harshad16) [inline README.rst:57]:
```suggestion
- The package size, number of downloads, maintainers and last release date from `PyPI <https://pypi.org/>`_.
```

moving the links part to new line causes rendering issue.

Comment (mayaCostantini) [reply README.rst:57]: Sorry about the noise, I have trouble rendering it locally.

Comment (harshad16) [reply README.rst]: That is totally fine.
thanks

**Pull request merged**